go/client/logclient_test.go

```diff
@@ -258,7 +259,9 @@ func TestAddChainRetries(t *testing.T) {
 		started := time.Now()
 		sct, err := client.AddChain(deadline, chain)
 		took := time.Since(started)
-		if math.Abs(float64(took-test.expected)) > float64(leeway) {
+		delta := math.Abs(float64(took - test.expected))
+		ratio := delta / float64(test.expected.Nanoseconds())
```

Comment:
Why are you only calling Nanoseconds() here, and not on any of the other Durations? I believe it's effectively a no-op here anyway.